src/transform/transform.ts

```diff
@@ -483,19 +526,51 @@ function createResponseType(codeModel: CodeModel, group: OperationGroup, op: Ope
     if (codeModel.language.go!.pollerTypes === undefined) {
       codeModel.language.go!.pollerTypes = new Array<PollerInfo>();
     }
+    let type = 'HTTP';
```

Comment:
Took me a bit to figure out what this chunk of code is doing, might be worth a comment.  Also, I think you can simplify it a bit.
```ts
let type = 'HTTP';
if (isSchemaResponse(firstResp) && firstResp.schema.language.go!.responseType.value) {
  type = firstResp.schema.language.go!.responseType.value;
}
```

Reply:
Will add a comment about this

Reply:
Done